.gitignore

```diff
@@ -110,12 +110,9 @@ kubernetes.tar.gz
 # generated files in any directory
 # TODO(thockin): uncomment this when we stop committing the generated files.
 #zz_generated.*
-zz_generated.openapi.go
```

Comment:
Is the actual openapi file missing?

Reply:
@sttts @soltysh said I should add it in a commit to the end of this rebase.